internal/provider/user_resource.go

```diff
@@ -115,8 +115,9 @@ func (r *userResource) Schema(ctx context.Context, _ resource.SchemaRequest, res
 				},
 			},
 			"namespace_accesses": schema.ListNestedAttribute{
-				Description: "The list of namespace accesses.",
-				Optional:    true,
+				Description:        "The list of namespace accesses.",
+				Optional:           true,
+				DeprecationMessage: "Use the `temporalcloud_user_namespace_access` resource to manage namespace access.",
```

Comment:
when will we remove namespace_accesses? how long will it remain as deprecated?

Reply:
I don't think there's much harm in keeping it around to avoid breaking our existing users. I definitely don't want to break people by removing it too quickly, even if we are in a pre-release.